k8s/edge/nginx-jwt/basexx.lua

```diff
@@ -0,0 +1,167 @@
+------------------------------------------------------------------------------------------------------------------------
+-- util functions
+------------------------------------------------------------------------------------------------------------------------
```

Comment:
I would much rather have it git cloned from a tag than stored a replica locally. So, at least Github can track dependencies in case a security issue identified.